Description: Miscellaneous spelling fixes
 A few were fixed by upstream already, so the PR below in the
 "Forwarded" field is smaller.
Author: Andreas Hasenack <andreas@canonical.com>
Forwarded: https://github.com/pmem/pmdk/pull/2841
Applied-Upstream: https://github.com/krzycz/pmdk/commit/7876719b3bbf4dfc2d655b011773ae50229010f0
Last-Update: 2018-04-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/libpmempool/libpmempool.c
+++ b/src/libpmempool/libpmempool.c
@@ -238,7 +238,7 @@
 	 */
 	if (util_flag_isset(args->flags, PMEMPOOL_CHECK_DRY_RUN) &&
 			args->backup_path != NULL) {
-		ERR("dry run does not allow to perform backup");
+		ERR("dry run does not allow one to perform backup");
 		errno = EINVAL;
 		return NULL;
 	}
--- a/doc/generated/pmempool-info.1
+++ b/doc/generated/pmempool-info.1
@@ -145,7 +145,7 @@
 Print pool's internal data in mixed format which consists of hexadecimal
 dump of header's data and parsed format displayed in human\-readable
 format.
-This allows to see how data is stored in file.
+This allows one to see how data is stored in file.
 .PP
 \f[C]\-s,\ \-\-stats\f[]
 .PP
--- a/doc/pmempool/pmempool-info.1.md
+++ b/doc/pmempool/pmempool-info.1.md
@@ -138,7 +138,7 @@
 
 Print pool's internal data in mixed format which consists of hexadecimal dump of
 header's data and parsed format displayed in human-readable format. This
-allows to see how data is stored in file.
+allows one to see how data is stored in file.
 
 `-s, --stats`
 
--- a/src/test/tools/pmemspoil/README
+++ b/src/test/tools/pmemspoil/README
@@ -4,8 +4,8 @@
 
 This directory contains a simple application for corrupting pool files.
 
-This application may be used for testing purposes. One allows to modify every
-single field in all structures used by Persistent Memory Development Kit.
+This application may be used for testing purposes. It allows one to modify
+every single field in all structures used by NVM Library.
 
 Usage:
 	$ pmemspoil <file> <field>=<value>...
--- a/src/examples/libpmemobj/array/README
+++ b/src/examples/libpmemobj/array/README
@@ -3,7 +3,7 @@
 This is examples/libpmemobj/array/README.
 
 This directory contains an example application implemented using libpmemobj.
-The array example allows to perform basic operations like allocation,
+The array example allows one to perform basic operations like allocation,
 reallocation and de-allocation on persistent array. User can choose
 size of new or reallocated array. There are also 3 types of elements which
 are supported: int, PMEMoid and TOID with structure containing int variable.
